broker-daemon/models/block-order.js

```diff
@@ -160,7 +160,24 @@ class BlockOrder {
    */
   fail () {
     this.status = BlockOrder.STATUSES.FAILED
+    return this
+  }
+
+  /**
+   * Move the block order to a completed status
+   * @return {BlockOrder} Modified block order instance
+   */
+  complete () {
+    const activeOrders = this.openOrders.filter(o => o.status !== BlockOrder.STATUSES.COMPLETED)
+    const activeFills = this.fills.filter(f => f.status !== BlockOrder.STATUSES.COMPLETED)
```

Comment:
See comment above: can't a block order be completed even if a fill was rejected (shouldn't we even expect that case)?